GeosCore/convection_mod.F90

```diff
@@ -304,7 +304,9 @@ SUBROUTINE DO_CONVECTION( Input_Opt,  State_Chm, State_Diag, &
           DO S = 1, State_Diag%Map_WetLossConv%nSlots
              NW = State_Diag%Map_WetLossConv%slot2id(S)
              DO L = 1, State_Grid%NZ
-                State_Diag%WetLossConv(I,J,L,S) = Diag38(L,NW)
+!                State_Diag%WetLossConv(I,J,L,S) = Diag38(L,NW)
+                State_Diag%WetLossConv(I,J,L,S) = Diag38(L,NW) &
+                                                / AREA_M2
```

Comment:
There are not currently plans to change wet loss convection diagnostic units from kg/s to kg/m2/s in the standard model. I can put this in MODEL_GEOS blocks, or would you like to petition to get the standard units changed?

Reply:
I very much prefer normalized units for GEOS because of the CS to lat-lon conversion. Having this update in MODEL_GEOS blocks is fine with me.